test/test-service-external-db.js

```diff
@@ -19,7 +18,7 @@ test('Service environment variable manipulation', function(t) {
     server.start(function onStart(err) {
       tt.ifError(err, 'server started successfully');
       Service = server.models.ServerService;
-      tt.equal(Service.dataSource, db);
+      tt.equal(Service.dataSource.connector.name, 'sqlite3');
```

Comment:
@rmg Not sure what you mean by default dataSource. AFAIK Loopback doesnt have a concept of a default datasource... it has to be explicitly specified in the model-config.json.